packages/quantic/force-app/main/default/lwc/quanticPill/quanticPill.js

```diff
@@ -34,7 +34,7 @@ export default class QuanticPill extends LightningElement {
   @api actionName;
 
   deselect() {
-    this.dispatchEvent(new CustomEvent('deselect'));
+    this.dispatchEvent(new CustomEvent('quantic__deselect'));
```

Comment:
Search for where the `<c-quantic-pill>` is used to see where possibly we have the `ondeselect={}`